tools/internal_ci/macos/pull_request/grpc_basictests_dbg.cfg

```diff
@@ -15,17 +15,11 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/macos/grpc_run_tests_matrix.sh"
-gfile_resources: "/bigstore/grpc-testing-secrets/gcp_credentials/GrpcTesting-d0eeee2db331.json"
-timeout_mins: 240
+build_file: "grpc/tools/internal_ci/macos/grpc_ios_binary_size.sh"
+timeout_mins: 60
 action {
   define_artifacts {
     regex: "**/*sponge_log.xml"
     regex: "github/grpc/reports/**"
   }
 }
-
-env_vars {
-  key: "RUN_TESTS_FLAGS"
```

Comment:
I wonder if we should leave this?

Reply:
Oh, I think we have to unfortunately. It can be set to anything.